bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/KeyContent.java

```diff
@@ -31,13 +34,10 @@ public KeyContent(KeyKeeper keeper) {
 	}
 
 	public byte[] get() throws LicensingException {
-		try (InputStream in = keeper.productPublicKey(); ByteArrayOutputStream out = new ByteArrayOutputStream()) {
-			int acquired = -1;
-			byte[] buffer = new byte[1024];
-			while ((acquired = in.read(buffer)) != -1) {
-				out.write(buffer, 0, acquired);
-			}
-			return out.toByteArray();
+		try (InputStream in = keeper.productPublicKey();
+				BufferedReader reader = new BufferedReader(new InputStreamReader(in, StandardCharsets.US_ASCII))) {
+			String content = reader.lines().collect(Collectors.joining()); // filter line-delimiters out
```

Comment:
format as all stream operations are formatted here: 
```
stream//
  .non-terminal //
  .non-terminal //
  .terminal //
  .otherThings;
```
There is no need for a variable for the result. Return inlined.

Reply:
Done.